### PR TITLE
Release/v1.0.1

### PR DESCRIPTION
**Pull request by ellendejong**
- README update. #2 
- Update executor settings in config to reduce the load on the slurm scheduler. #2 
- Update submodule NextflowModules. Not reviewed previously.
- General clean-up of code. Not reviewed previously. 
    - Indentation changes.
    - Refactor of `file` to `path`.

[Azure 99960 - release branch test](https://dev.azure.com/umcu/LAB-Bioinformatics-Genetics/_workitems/edit/99960) 
[Azure 99698 - initial test executor settings](https://dev.azure.com/umcu/LAB-Bioinformatics-Genetics/_workitems/edit/99698)
[Azure feature 99330](https://dev.azure.com/umcu/LAB-Bioinformatics-Genetics/_workitems/edit/99330)

### CODE REVIEW

File: run_nextflow_mip.sh [hunk 1]
```diff
@@ -38,11 +38,11 @@ module load Java/1.8.0_60
...
 
-    echo "Running Nextflow clean"
-    /hpc/diaggen/software/tools/nextflow clean -f -k -q
-
     echo "Zip work directory"
-    zip -r -m -q work.zip work
+    find work -type f | egrep "\.(command|exitcode)" | zip -@ -q work.zip
+
+    echo "Remove work directory"
+    rm -r work
```
Comment (FiniDG) [inline run_nextflow_mip.sh:45]: first the work directory was zipped, now removed. is there a reason for this?

File: run_nextflow_mip.sh [hunk 2]
```diff
@@ -38,11 +38,11 @@ module load Java/1.8.0_60
 if [ \$? -eq 0 ]; then
     echo "Nextflow done."
 
-    echo "Running Nextflow clean"
-    /hpc/diaggen/software/tools/nextflow clean -f -k -q
```
Comment (FiniDG) [inline run_nextflow_mip.sh:42]: clean command not needed anymore?

File: run_nextflow_mip.sh (hunk 1 above)
Comment (ellendejong) [reply run_nextflow_mip.sh:45]: Yes. The work directory contains all input, outputs and command files of tasks. This is quite a large directory.
The find command on line 42 searches for relevant files and creates a zip with the things that need to be retained. 
Afterwards, the directory is removed.

Comment (FiniDG) [reply run_nextflow_mip.sh:45]: :+1:

File: run_nextflow_mip.sh (hunk 2 above)
Comment (ellendejong) [reply run_nextflow_mip.sh:42]: I believe cleaning via nextflow is replaced with the find command on line 42.

Comment (FiniDG) [reply run_nextflow_mip.sh:42]: check. that explains both my comments I guess :+1:

**Review by FiniDG — DISMISSED** [review]
We talked separately about the many changes in the modules, because those cannot be added to this pull. All my questions were clearly answered about these changes (relevant to the DxNextflowMIP).
The comments were also clearly answered, and the executor settings were already discussed within the other DxNextflow pipelines.

Approval :+1:

**Review by FiniDG — APPROVED** [review]
Discussed in Teams for a bit of clarification. Is an update related to the nextflow modules repo. All is explained clearly.

**Pull request merged**